Automattic/domain-services-client
# Hide internal methods from documentation

This PR hides the methods that won't be promoted to the resellers, using `@internal` and changing their visibility where appropriate.

## Testing

- `composer install`
- `./build.sh clean-docs && ./build.sh build-only`
- Inspect the docs generated at `./docs`:
  - Ensure all private methods are not generated
  - Ensure that all methods we want to promote to the resellers are visible
